frontend/src/metabase/common/components/Table/Table.tsx

```diff
@@ -60,7 +55,7 @@ export function Table<Row extends BaseRow>({
 }: TableProps<Row>) {
   return (
     <>
-      <table {...rest} className={CS.Table}>
+      <table className={CS.Table} {...rest}>
```

Comment:
Let's adjust this so that we can expect a `className` to be passed in and added to `CS.Table`, rather than potentially override it